tests/test_modeling_common.py

```diff
@@ -1782,6 +1782,10 @@ def test_resize_tokens_embeddings(self):
             self.assertEqual(type_model_embed_pre_resize, type_model_embed_post_resize)
             # Check that the model can still do a forward pass successfully (every parameter should be resized)
             model(**self._prepare_for_class(inputs_dict, model_class))
+            # Check that added embeddings mean is close to the old embeddings mean
+            old_embeddings_mean = torch.mean(model_embed.weight.data[:model_vocab_size, :], axis=0)
+            new_embeddings_mean = torch.mean(model_embed.weight.data[-10:, :], axis=0)
```

Comment:
```suggestion
            old_embeddings_mean = torch.mean(model_embed.weight.data[:-10, :], axis=0)
            new_embeddings_mean = torch.mean(model_embed.weight.data[-10:, :], axis=0)
```
For consistency, let's use -10 in both places!